controllers/progressivesync_controller.go

```diff
@@ -99,6 +103,15 @@ func (r *ProgressiveSyncReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 		return ctrl.Result{Requeue: true}, nil
 	}
 
+	hashValue, err := r.calculateHashValue(ctx, &ps)
+	if err != nil {
+		log.Info("Failed to generate the hash value from the ApplicationSet spec")
```

Comment:
that looks like a blocking error? or do we have a fallback plan?

Reply:
It is just a logging error for now. Once we start working on the next feature, I think it will be more clear what we want to do here.

Reply:
I have now updated it to return the error we got above. I will resolve for now 🙂